src/bytesToUuid.js

```diff
@@ -13,29 +13,31 @@ function bytesToUuid(buf, offset) {
 
   const bth = byteToHex;
 
-  // join used to fix memory issue caused by concatenation: https://bugs.chromium.org/p/v8/issues/detail?id=3175#c4
-  return [
-    bth[buf[i + 0]],
-    bth[buf[i + 1]],
-    bth[buf[i + 2]],
-    bth[buf[i + 3]],
-    '-',
-    bth[buf[i + 4]],
-    bth[buf[i + 5]],
-    '-',
-    bth[buf[i + 6]],
-    bth[buf[i + 7]],
-    '-',
-    bth[buf[i + 8]],
-    bth[buf[i + 9]],
-    '-',
-    bth[buf[i + 10]],
-    bth[buf[i + 11]],
-    bth[buf[i + 12]],
-    bth[buf[i + 13]],
-    bth[buf[i + 14]],
-    bth[buf[i + 15]],
-  ].join('');
+  // Note: Be careful editing this code!  It's been tuned for performance
+  // and works in ways you may not expect. See https://github.com/uuidjs/uuid/pull/434
+  // `toLowerCase` used to fix memory issue caused by concatenation: https://bugs.chromium.org/p/v8/issues/detail?id=3175#c4
```

Comment:
Referring to this PR should suffice.  Readers can't get from here to 267 and the Chromium issue if they want to dive that deep.

```suggestion
```

Reply:
Ok, as you say…